vector/src/main/res/layout/fragment_room_detail.xml

```diff
@@ -98,7 +98,7 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         app:barrierDirection="top"
-        app:constraint_referenced_ids="composerLayout,notificationAreaView,failedMessagesWarningView" />
+        app:constraint_referenced_ids="composerLayout,notificationAreaView,failedMessagesWarningStub" />
```

Comment:
I think the other Barrier needs to update too

Reply:
It's already `failedMessagesWarningStub`, do you confirm it's wrong?
https://github.com/vector-im/element-android/blob/develop/vector/src/main/res/layout/fragment_room_detail.xml#L207

Reply:
Yes, it is.

Reply:
it is... wrong?